optimizely/event/event_factory.py

```diff
@@ -97,7 +100,18 @@ def create_log_event(
 
         event_params = event_batch.get_event_params()
 
-        return log_event.LogEvent(cls.EVENT_ENDPOINT, event_params, cls.HTTP_VERB, cls.HTTP_HEADERS)
+        region = user_context.region
+        if hasattr(region, 'value'):
+            region_str = region.value
+        elif region is None:
+            region_str = 'US'  # Default to US
+        else:
+            region_str = str(region)
```

Comment:
We can consider using a simple string type for region. The region will be handled in type-safe way when building into datafile. It looks like fallback to "US" for safety is good enough?

Reply:
@jaeopt by string type do you mean using Region.US instead of string 'US'?
That's what I have been suggesting further down the code. Just we don't contradict each other.